workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,381 @@
+    balls(turtle, "#fc045b", -x, y, 5) # Decorative balls
+    y = y + height
+
+```
+Not too difficult, is it ? Let's break this down.
+
+
+```python
+width = width - 20
+height = randint(20,30)
+```
+In each iteration, the width of the next rectangle gets reduced to a smaller size so that in the end we get a triangle kinda look for the entire leaf part (green portion). We assign a random value for the height as that helps to give the tree an irregular shape rather than a giving same height for all the rectangles. If you would like, you could reduce the width by a random integer too. 
+
+
+```python
+x = 0 - width/2
+```
+This is just to set the starting 'x' coordinate.
+
+```python
+box1(turtle, "#05A167", x, y, width, height) # Box for the leaves of tree
+```
+Now we draw one complete green color filled box for first layer of flowers.
+![alt text](https://cloud-4ipo9natc.vercel.app/5image4.png) 
+
+```python
+balls(turtle, "#fc045b", x, y, 5) # Decorative balls
+```
+Since the turtle is back on the left end of the rectangle, this statement draws a small ball of radius 5 with a shade of red filled in it. 
+![alt text](https://cloud-4ipo9natc.vercel.app/6image5.png) 
+Now we need one more tiny ball on the right end of the christmas tree right ? 
+
+```python
+balls(turtle, "#fc045b", -x, y, 5)
+```
+So we draw another ball with the same dimentions but we pass a different value for x coordinate. We pass ******-x****** instead of ******x******. This helps to position the ball on the right end instead of the left. 
+![alt text](https://cloud-4ipo9natc.vercel.app/7image6.png) 
+```python
+y = y + height
+```
+Finally, the above line helps to move the turtle up to the top of each rectangle so that the next rectangle it draws will be stacked on top of the one before it. 
+Repeating the above step over and over again will give us recatangles stacked on top of each other such that ones on top are slightly smaller then the one directly under it. We also have tiny red balls on either side.
+![alt text](https://cloud-4ipo9natc.vercel.app/8image7.png) 
+
+Next,  Lets create a star on top of the tree.
+Note that once the rectangle drawing is done, the turtle is already on top of the tree. 
+```python
+#Star on the top 
+star(turtle, "#e5f614", 0.4, y, 20)
+```
+The above code will draw a yellow star on top of the tree. You can tweak the parameters to see how the positioning and size varies. 
+![alt text](https://cloud-4ipo9natc.vercel.app/9image8.png) 
+
+Wow ! The tree is done. Now for the final touch, the greetings . 
+
+```python
+#Christmas wishes
+penup()
+color("#ff59ac")
+goto(-250, -250)
+write("Merry Christmas !!!", font=("Calibri", 50, "bold"))
+```
+I believe the code above is pretty self explanatory. We used the write method to have a text be written on the screen.
+Try changing the parameters of the ******goto()****** , ******color()****** and ******write()****** function to see the differences. 
+
```

Comment:
```suggestion
I believe the code above is pretty self explanatory. We used the write method to have a text be written on the screen.
Try changing the parameters of the ******goto()****** , ******color()****** and ******write()****** function to see the differences. 

```

Reply:
am not sure what your suggestion is about line 294-296 . Should I be explaining that potion as well ?

Reply:
if your mentioning about the over use of *, then I shall change that and make back ticks instead.